migration/mule/rest-api-with-apikit/README.md

```diff
@@ -0,0 +1,44 @@
+# REST API with APIkit
+
+This example illustrates an API implementation with Integration Studio which was created using a OAS2 swagger file. A mock backend is used in the implementation, whereas the user is provided the capability of using a backend of choice instead of the one provided.
+
+### Assumptions ###
+
+This document assumes that you are familiar with the [Integration Studio](https://ei.docs.wso2.com/en/latest/micro-integrator/develop/WSO2-Integration-Studio/). To increase your familiarity with Studio, consider completing one or more [Integration Studio Tutorials](https://ei.docs.wso2.com/en/latest/micro-integrator/use-cases/integration-use-cases/). Further, this example assumes that you have a basic understanding of [EI flows](https://ei.docs.wso2.com/en/latest/micro-integrator/overview/key-concepts/).
+
+### Example Use Case ###
+
+This API mocks endpoints for customer entity.
+
+### Set Up and Run the Example ###
+
+Follow the procedure below to run and test the functionality in Integration Studio.
+
+1. Start WSO2 Integration Studio ([Installing WSO2 Integration Studio](https://ei.docs.wso2.com/en/latest/micro-integrator/develop/installing-WSO2-Integration-Studio/)).
+2. In your menu in Studio, click the **File** menu. In the File menu select the **Import...** item.
+3. In the Import window select the **Existing WSO2 Projects into workspace** under **WSO2** folder.
+4. Browse and select the file path to the downloaded sample of this Github project
+(``integration-studio-examples/migration/mule/rest-api-with-apikit``) and click **Finish**.
+5. Open the **RestApiWithApikit.xml** under **rest-api-with-apikit/RestApiWithApikit/src/main/synapse-config/api** 
+directory.
+
+`image of the design here`
+
+6. The **RestApiWithApikit.xml** is the graphical view of the resources of the used API.
```

Comment:
Shall we mention that this API is generated by the swagger file attached in this project (at location x)?

Reply:
The way to generate the API should come to this step. 

[Creating a REST API](https://ei.docs.wso2.com/en/latest/micro-integrator/develop/creating-artifacts/creating-an-api/)